src/homework3/task2.py

```diff
@@ -0,0 +1,16 @@
+"""
+List practice
+
+1. Используйте генератор списков чтобы получить следующий: ['ab', 'ac', 'ad', 'bb', 'bc', 'bd'].
+2. Используйте на предыдущий список slice чтобы получить следующий: ['ab', 'ad', 'bc'].
+3. Используйте генератор списков чтобы получить следующий ['1a', '2a', '3a', '4a'].
+4. Одной строкой удалите элемент  '2a' из прошлого списка и напечатайте его.
+5. Скопируйте список и добавьте в него элемент '2a' так
+чтобы в исходном списке этого элемента не было.
+"""
+
+list_for_task1 = [chr1 + chr2 for chr1 in 'ab' for chr2 in 'bcd']
+list_for_task2 = list_for_task1[::2]
+list_for_task3 = [str(chr_) + 'a' for chr_ in range(1, 6)]
```

Comment:
1. doesn't solve task in the description. 
2. '1234' - not addressed review issue